pkg/cmd/roachtest/tests/acceptance.go

```diff
@@ -107,6 +107,14 @@ func registerAcceptance(r registry.Registry) {
 				requiresLicense: true,
 				disallowLocal:   true,
 			},
+			{
+				name:          "benchmark-leasing",
+				numNodes:      3,
+				disallowLocal: true,
+				nodeRegions:   []string{"us-west1-c", "us-east1-c", "australia-southeast1-a"},
+				fn:            runAcceptanceMultiRegionBenchmarkLeasing,
+				timeout:       time.Hour,
```

Comment:
Can we move this test to a non-acceptance location? The test is verifying a new feature, so it shouldn't really be an "acceptance" test -- acceptance tests are supposed to be smoke tests.